criu/files-reg.c

```diff
@@ -13,8 +13,11 @@
 #include <sys/capability.h>
 #include <sys/mount.h>
 #include <elf.h>
+#include <linux/fiemap.h>
+#include <linux/fs.h>
```

Comment:
@featherchen Would you be able to move these two include before `#include <sys/mount.h>`?
This should fix the CI errors we see for Arch Linux.

Reply:
OK, I have done this change.
However, it seems somehow cause more errors happends(looks like related to `<sys/mount.h>)`